.github/workflows/main.yaml

```diff
@@ -0,0 +1,64 @@
+# This file lints the code and runs tests.
+name: CI
+
+on:
+  pull_request:
+  push:
+
+env:
+  COMPOSER_FLAGS: "--ansi --no-interaction --no-progress --no-suggest --prefer-dist"
+
+jobs:
+  tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php:
+          - '8.0'
+          - '7.4'
+          - '7.3'
+          - '7.2'
+          - '7.1'
+          - '7.0'
+          - '5.6'
+        include:
+          - php: '7.0'
+            cs_fixer: true
+    name: 'Check with PHP ${{ matrix.php }}'
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Nix
+        uses: cachix/install-nix-action@v12
+
+      - name: Set up Nix cache
+        uses: cachix/cachix-action@v8
+        with:
+          name: fossar
```

Comment:
Is this correct?

Reply:
The way Nix works, it takes build instructions for packages and builds them. It can also substitute the build result from a cache to reduce work. But since these are custom packages, they are not built by the official build servers and we need to use our own cache. I added a comment with a link to the repository that builds it.